### PR TITLE
Initialize land use database in PostGIS

### LINKED ISSUE #1: QGIS tool for initializing the land use plan database in PostGIS
# Button for PostGIS-connection

Land use planner needs to establish a connection to PostGIS database and run the initialization sql for land use planning database. At the moment there are 2 possible database models to run: [a detailed land use plan database model](https://github.com/GispoCoding/qaava/tree/master/asemakaavan-tietomalli) and a general plan database model.

## User story:

1. Land use planner opens QGIS and downloads the QAAVA plugin 
2. Land use planner clicks the designated button for initialization of the land use database 
3. User chooses which land use plan database they want (2 options at the moment, slightly different models)
4. Users connects to existing PostGIS database 
5. With "run" the database is set-up

## Questions:

-  If the user do not have their own PostGIS set-up, we can offer maybe SAAS-service or options for possible service providers?
- Does the user need to have an ability to add some needed information to the database with a form? And what information?
- Does the user have the possibility to add other users rights to the database?
- After the set-up, do we need a button that opens the needed datasets with relations and forms to the QGIS project? In this case, maybe a ready made QGIS profile with connections to the right database.

### PR DESCRIPTION
**Pull request by Joonalai**
This PR will resolve #1. 

WIP: 
- [ ] The schema for general land use planning is missing
- [ ] Versioning of the schema is not yet implemented

### CODE REVIEW

**Comment by Joonalai** [comment]
@mvaaltola would you like to do the code review for the PR and tell your comments about it?

File: Qaava/database_tools/gui/db_tools_initializer_dialog.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+        self.populate_dbComboBox()
+        self.populate_dmComboBox()
+
+        self.buttonBox.button(QDialogButtonBox.Ok).setText(tr(u"Run"))
+
+        self.agreedCheckBox.setChecked(False)
+        self.on_agreedCheckBox_stateChanged()
+
+    def populate_dbComboBox(self):
```
Comment (mvaaltola) [inline Qaava/database_tools/gui/db_tools_initializer_dialog.py:36]: New connections aren't added until plugin reload. Suggestion: add a refresh button to the UI to reload available connections. I'll create a separate issue for this as IMO it's more of a feature request than a fix.

**Comment by mvaaltola** [comment]
Everything looks good, great work. Merging this and creating a feature request issue for the db connection reload commented above.

File: Qaava/database_tools/gui/db_tools_initializer_dialog.py (hunk 1 above)
Comment (Joonalai) [reply Qaava/database_tools/gui/db_tools_initializer_dialog.py:36]: Agreed!

**Pull request merged**